### ISSUE TITLE
Wide band model

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#722]
```
Please complete the following lines...

Application Version: 5.3.10
SVN Revision Number: 3193 (?)
Compile Date: Feb 03
Operating System: Windows XP

Describe details of the issue below:

When testing the adiabatic surface temperature output option, the
calculated data deviates from the "default" calculation. A check of the HHR
and rad_loss compared to wide_band_model = .false. shows a large
fluctuation of the values.

```

Original issue reported on code.google.com by `jsandstrom42` on 2009-05-18 09:18:48

<hr>
- _Attachment: [wbm.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-744/comment-0/wbm.fds)_

**Comment by gforney** [comment]
```
I am not sure what you mean. What is the "calculated data" and what is the "default"

calculation.
```

Original issue reported on code.google.com by `mcgratta` on 2009-05-18 11:27:59

**Comment by gforney** [comment]
```
The calculated data is the output data (calculated by FDS).

Default refers to all the submodels such as radiation_angles where no changes have
been made. No compilation changes have been made.
```

Original issue reported on code.google.com by `jsandstrom42` on 2009-05-18 12:04:15

**Comment by gforney** [comment]
```
Kevin,
This has to deal with KFST4_ALTERNATIVE computation having KAPPA*UII added.
So, currently RADIATIVE_FRACTION > 0 is not kosher with WIDE_BAND_MODEL.


```

Original issue reported on code.google.com by `shostikk` on 2009-05-18 13:08:54

**Comment by gforney** [comment]
```
I have just committed a new source fixing the bug. It was the inclusion of KAPPA*UII
in the KFST4_ALTERNATIVE. Now that is multipled by BFF.

Joakim, please test with the new version (can you compile yourself?), and verify.

```

Original issue reported on code.google.com by `shostikk` on 2009-05-18 13:46:35

**Comment by gforney** [comment]
```
I'm not sure on how to compile but I guess I have to learn. Where can I get the new
version? or should I go into the code and change?

Do I need both C++ and fortran 90 to compile?
```

Original issue reported on code.google.com by `jsandstrom42` on 2009-05-18 14:01:44

**Comment by gforney** [comment]
```
To compile you need to "check out" the repository. Read the wiki called "Accessing 
the SVN or Subversion Repository". Then you need a fortran and c compiler. There are

wiki instructions as well. If you're not familiar with compiling, you might want to

wait on the next release.
```

Original issue reported on code.google.com by `mcgratta` on 2009-05-18 17:02:06

**Comment by gforney** [comment]
```
I'll see if I learn, I at least have the ambition...

Thank you for the quick response from all of you.
```

Original issue reported on code.google.com by `jsandstrom42` on 2009-05-18 18:01:31

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `mcgratta` on 2009-06-09 12:41:51
- **Labels removed**: Milestone-Release_5

**Comment by gforney** [comment]
```
Tested the same file with the 5.4.1 exefile from mr Rogsch. With the MPI version all
worked fine (it seems) but with the openMP version I got the "same" results as
started this issue. Should I start a new Issue or is this OK?

It is possible to use the same file as earlier.

New version info:

 Compilation Date : Fr, 04 Sep 2009
 Version          : 5.4.0 Serial
 OpenMP Enabled
 Number of OpenMP threads:   2
 SVN Revision No. : 4675

```

Original issue reported on code.google.com by `jsandstrom42` on 2009-10-01 14:00:53

**Comment by gforney** [comment]
```
If you are getting different results using the OpenMP version than you obtain with 
the non-OpenMP version, start a new version. Remember to post a simple input file 
and tell us what versions you are using.
```

Original issue reported on code.google.com by `mcgratta` on 2009-10-01 14:19:31

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `mcgratta` on 2009-11-23 22:23:44
